app/src/services/transfer.ts

```diff
@@ -40,8 +40,8 @@ export const getErc20TokenContract = (
   env: Snowbridge.environment.SnowbridgeEnvironment,
 ): string => {
   switch (token.id) {
-    case 'weth':
-      return env.locations[0].erc20tokensReceivable[0].address
+    case 'weth' | 'veth':
+      return env.locations[0].erc20tokensReceivable[token.id].address
```

Comment:
Here seems to be an error.
```suggestion
    case 'weth':
    case 'veth':
      return env.locations[0].erc20tokensReceivable[NUMBER].address // TODO: this now takes a number and not a string. So we need to change it
```

Reply:
So for WETH it should be index = 0

Reply:
I have a better fix for this 👍